src/components/Remark.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019-2020 @paritytech/bridge-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import Keyring from '@polkadot/keyring';
+import { Codec } from '@polkadot/types/types';
+import { u8aToHex } from '@polkadot/util';
+import React, { useState } from 'react';
+import { Button, Container, Input } from 'semantic-ui-react';
+import styled from 'styled-components';
+
+import { useApiSourcePromiseContext } from '../contexts/ApiPromiseSourceContext';
+import { useApiTargetPromiseContext } from '../contexts/ApiPromiseTargetContext';
+import useLoadingApi from '../hooks/useLoadingApi';
+interface Props {
+  className?: string;
+  targetChain: string;
+}
+
+const Remark = ({ className, targetChain }: Props) => {
+  const { api: sourceApi } = useApiSourcePromiseContext();
+  const { api: targetApi } = useApiTargetPromiseContext();
+
+  const areApiLoading = useLoadingApi();
+  const [remarkInput, setRemarkInput] = useState('0x');
+
+  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setRemarkInput(event.target.value);
+  };
+
+  async function sendMessageRemark() {
+    const keyring = new Keyring({ type: 'sr25519' });
+    const account = keyring.addFromUri('//Alice');
+
+    const remarkCall = await targetApi.tx.system.remark(remarkInput);
```

Comment:
The expectation seems to be that `remarkInput` is a `0x`-prefixed hex bytes string, what if it's not correct? Will it fail here or be converted?

Reply:
actually i think that if it's not a `0x` prefixed hex , the input will be consider as ascii right? the call does not fail if you provide a not 0x fyi

Reply:
Yeah, that was my expectation as well, but just wanted to confirm that.